src/components/button/index.tsx

```diff
@@ -97,7 +101,8 @@ class Button extends PureComponent<Props, ButtonState> {
   }
 
   getLabelColor() {
-    const {link, linkColor, outline, outlineColor, disabled, color: propsColor} = this.props;
+    const {linkColor, outline, outlineColor, disabled, color: propsColor} = this.props;
+    const link = this.isLink;
```

Comment:
The name should be 'isLink'.

Reply:
I was just keeping the old name, but I guess you're right, fixed